client-id/handler_test.go

```diff
@@ -205,3 +205,20 @@ func Test(t *testing.T) {
 		})
 	}
 }
+
+func Test_Options(t *testing.T) {
+	options := []client_id.Option{
+		client_id.WithStore(store.NewLocal()),
+		client_id.WithHeaderExtractor("X-Client-ID"),
+		client_id.WithRequired(),
```

Comment:
Do I interpret this correctly? 
1. We set clientID to required
2. We don't send a clientID header since we do a OPTIONS request
3. Validate that it ignores options

Reply:
yes